scss/partials/_dialog.scss

```diff
@@ -160,6 +160,73 @@
   }
 }
 
+.jsxc-tabbed-area figure {
+  clear: both;
+  display: block;
+  margin-left: 0;
+  max-height: 500px;
+  max-width: 500px;
+  min-height: 500px;
+  min-width: 500px;
+  overflow: auto;
+}
+
+.jsxc-tabbed-area > input,
+.jsxc-tabbed-area figure > section { display: none; }
+
+.jsxc-tabbed-area figure>section {
+  background: var(--jsxc-color-main-background);
+  border: 1px solid var(--jsxc-color-border);
+  color: var(--jsxc-color-text-light);
+  letter-spacing: 0.3px;
+  line-height: 1.5em;
+  margin-top: 0 !important;
```

Comment:
The use of important often indicates some issues with the html or css structure.